app/src/main/java/com/framgia/wsm/data/model/Notification.java

```diff
@@ -19,11 +19,19 @@
     private String mContent;
     @Expose
     @SerializedName("status")
-    private int mStatus;
+    private boolean isRead;
```

Comment:
WARNING: [AvoidFieldNameMatchingMethodName] Field isRead has the same name as a method
https://pmd.github.io/pmd-5.5.1/pmd-java/rules/java/naming.html#AvoidFieldNameMatchingMethodName